app/Http/Controllers/ViewAssetsController.php

```diff
@@ -181,15 +181,17 @@ public function getRequestAsset($assetId = null)
         // If it's already requested, cancel the request.
         if ($asset->isRequestedBy(Auth::user())) {
             $asset->cancelRequest();
+            $asset->decrement('requests_counter', 1);
+            
```

Comment:
Is requests_count the number of times it's ever been requested or the number of times it's currently requested?

Reply:
Ever been requested - unless a user actively cancels that request